autotest/src/test/kotlin/no/nav/familie/tilbake/e2e/autotest/Saksbehandler.kt

```diff
@@ -0,0 +1,149 @@
+                under4rettsgebyr = under4rettsgebyr,
+                muligforeldelse = muligforeldelse
+        )
+    }
+
+    fun opprettKravgrunnlag(
+            status: KodeStatusKrav,
+            @Max(29)
+            antallPerioder: Int,
+            under4rettsgebyr: Boolean,
+            muligforeldelse: Boolean
+    ) {
+        assertTrue(gjeldendeBehandling != null && gjeldendeBehandling?.fagsystem != null,
+                             "Fagsystem ikke definert. Opprett behandling først eller bruk opprettKravgrunnlagUtenBehandling")
+        assertTrue(gjeldendeBehandling != null && gjeldendeBehandling?.ytelsestype != null,
+                             "Ytelsestype ikke definert. Opprett behandling først eller bruk opprettKravgrunnlagUtenBehandling")
+        assertTrue(gjeldendeBehandling != null && gjeldendeBehandling?.eksternFagsakId != null,
+                             "EksternFagsakId ikke definert. Opprett behandling først eller bruk opprettKravgrunnlagUtenBehandling")
+        assertTrue(gjeldendeBehandling != null && gjeldendeBehandling?.eksternBehandlingId != null,
+                             "EksternBehandlingId ikke definert. Opprett behandling først eller bruk opprettKravgrunnlagUtenBehandling")
+
+        val request = opprettKravgrunnlagBuilder.opprettKravgrunnlag(
+                status = status,
+                fagområde = gjeldendeBehandling?.fagsystem!!,
+                ytelsestype = gjeldendeBehandling?.ytelsestype!!,
+                eksternFagsakId = gjeldendeBehandling?.eksternFagsakId!!,
+                eksternBehandlingId = gjeldendeBehandling?.eksternBehandlingId!!,
+                kravgrunnlagId =  gjeldendeBehandling?.kravgrunnlagId,
+                vedtakId =  gjeldendeBehandling?.vedtakId,
+                antallPerioder = antallPerioder,
+                under4rettsgebyr = under4rettsgebyr,
+                muligforeldelse = muligforeldelse
+        )
+        gjeldendeBehandling!!.kravgrunnlagId = request.detaljertKravgrunnlag?.kravgrunnlagId
+        gjeldendeBehandling!!.vedtakId = request.detaljertKravgrunnlag?.vedtakId
+        familieTilbakeKlient.opprettKravgrunnlag(kravgrunnlag = request)
+    }
+
+    fun hentBehandlingId(fagsystem: Fagsystem, eksternFagsakId: String, eksternBrukId: String?): String {
+        familieTilbakeKlient.hentFagsak(fagsystem, eksternFagsakId)?.behandlinger?.forEach {
+            if (it.eksternBrukId.toString() == eksternBrukId) {
+                gjeldendeBehandling?.behandlingId = it.behandlingId.toString()
+                return it.behandlingId.toString()
+            }
+        }
+        throw Exception("Fantes ikke noen behandling med eksternBrukId $eksternBrukId på kombinasjonen eksternFagsakId $eksternFagsakId og fagsystem $fagsystem")
+    }
+
+    fun erBehandlingPåVent(behandlingId: String, venteårsak: Venteårsak): Boolean {
+        familieTilbakeKlient.hentBehandling(behandlingId)?.behandlingsstegsinfo?.forEach {
```

Comment:
familieTilbakeKlient.hentBehandling(behandlingId)?.behandlingsstegsinfo?.any {it.behandlingsstegstatus == Behandlingsstegstatus.VENTER && it.venteårsak == venteårsak}